backend/core/src/application/application.service.ts

```diff
@@ -0,0 +1,14 @@
+import { Injectable } from "@nestjs/common"
+import { InjectRepository } from "@nestjs/typeorm"
+import { Application } from "../entity/application.entity"
+import { Repository } from "typeorm"
+
+@Injectable()
+export class ApplicationService {
+  constructor(
+    @InjectRepository(Application) private readonly applicationRepo: Repository<Application>
```

Comment:
Can we use the same ActiveRecord-style pattern here as we do for the Listings?

Reply:
Done. @bencpeters I guess the same applies to user and auth modules?

Reply:
Yes